amundsen_application/static/js/components/SearchPage/SearchPanel/index.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { ResourceType } from 'interfaces/Resources';
+import { GlobalState } from 'ducks/rootReducer';
+import { DashboardSearchResults, TableSearchResults, UserSearchResults } from 'ducks/search/types';
+import { TABLE_RESOURCE_TITLE, USER_RESOURCE_TITLE } from 'components/SearchPage/constants';
+import AppConfig from 'config/config';
+
+import './styles.scss';
+
+export interface StateFromProps {
+  selectedTab: ResourceType,
+  tables: TableSearchResults;
+  dashboards: DashboardSearchResults;
+  users: UserSearchResults;
+}
+
+export interface OwnProps {
+  onTabChange: (tab: ResourceType) => void;
```

Comment:
Right now this relies on a `onTabChange` function being passed in from the `SearchPage` which updates the URL.

Once we switch from using URL to using redux actions, this should also be removed.